<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLoggedChores</class>
 <widget class="QWidget" name="UserLoggedChores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="weekLabel">
     <property name="styleSheet">
      <string>font-size: 30px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>This Week</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="daysLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="navLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>← Previous Week</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next Week →</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string>font-size: 18px; padding: 10px;</string>
     </property>
     <property name="text">
      <string>← Back to Home</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
